test/test_utils.py

```diff
@@ -338,3 +339,34 @@ def test_expired(self):
           }
         """)
         self.assertEqual(isExpired, True)
+
+
+class TestFromEnv(TestCase):
+
+    def setUp(self):
+        self.savedEnv = os.environ.copy()
```

Comment:
Why not use the mock library's `patch.dict` context manager in the tests?

https://docs.python.org/dev/library/unittest.mock.html#unittest.mock.patch.dict